core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -91,6 +92,9 @@ trait EntityStatisticsCacheMonitor extends LazyLogging {
   }
 
   def updateStatisticsCache(workspaceId: UUID, timestamp: Timestamp): Future[Unit] = {
+    // allow 80% of the per-workspace timeout to be spent calculating the attribute names
```

Comment:
I think it might be worth including a comment that we aren't putting a timeout on the other 20%